lib/modules/manager/npm/post-update/pnpm.spec.ts

```diff
@@ -25,56 +28,115 @@ describe('modules/manager/npm/post-update/pnpm', () => {
   });
   it('generates lock files', async () => {
     const execSnapshots = mockExecAll(exec);
-    fs.readFile = jest.fn(() => 'package-lock-contents') as never;
+    fs.readFile = jest
+      .fn()
```

Comment:
```suggestion
    fs.readFile
```

Thise are already mocked, so please use new syntax

Reply:
im not sure what you mean, i thought im already using the new syntax,
anyway im changing all these tests back as its not needed anymore after doing the Lazy approach..

Reply:
You should not overwrite functions with mocks, those cant be automatically reset, so use normal mosule mocking like we do on all other tests. 😉

Reply:
i submitted new code, im not sure what you mean, this is like my second time using JEST, so i'm not familiar with those phrasings.

do i still use bad mocks after my latest push?, if yes maybe help me make them better so i can learn how to do it, that's what i found when i researched jest mocks.

Reply:
I know, you are editing tests, which are old and not yet updated to use new mocking stuff we added for simplier testing. So iut's probably not your fault, you simply copied existing bad code😉

Reply:
can u give an example? or documentation on how to do things the right way? im new to this